src/blas/backends/cublas/CMakeLists.txt

```diff
@@ -42,7 +41,9 @@ set_target_properties(${LIB_OBJ} PROPERTIES
   POSITION_INDEPENDENT_CODE ON)
 
 target_link_libraries(${LIB_NAME} PUBLIC ${LIB_OBJ})
-
+if (USE_ADD_SYCL_TO_TARGET_INTEGRATION)
+  add_sycl_to_target(TARGET ${LIB_OBJ} SOURCES ${SOURCES})
```

Comment:
do we even have to specify the `SOURCES` here..? (not sure about the semantics with or without :D)

Reply:
hipSYCL doesn't need it, but it's the correct interface for portability, as e.g. ComputeCpp needs it AFAIK.